commit 5b3cc796183f7c653a702f81584daeea6783444d
Author: Jakub Kicinski <jakub.kicinski@netronome.com>
Date:   Fri Feb 24 11:43:37 2017 -0800

    geneve: lock RCU on TX path
    
    [ Upstream commit a717e3f740803cc88bd5c9a70c93504f6a368663 ]
    
    There is no guarantees that callers of the TX path will hold
    the RCU lock.  Grab it explicitly.
    
    Fixes: fceb9c3e3825 ("geneve: avoid using stale geneve socket.")
    Signed-off-by: Jakub Kicinski <jakub.kicinski@netronome.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1382_5b3cc796183f7c653a702f81584daeea6783444d.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1382_5b3cc796183f7c653a702f81584daeea6783444d.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
